### ISSUE TITLE
Cannot run/build the app (using Angular)

### ISSUE THREAD

**Issue by soyzamudio** [swellstores/swell-js#90]
I am building a store and using `swell-js` 3.19.6 and I have to modify the following `SnakeToCamelCase` type in `node_modules/swell-js/types/index.js` to get the app running locally:

Modified from:
```ts
type SnakeToCamelCase<S extends string> = S extends `${infer T}_${infer U}`
  ? `${T}${Capitalize<SnakeToCamelCase<U>>}`
  : S;
```
to:
```ts
export type SnakeToCamelCase<S extends any> = S extends `${infer T}_${infer U}`
  ? `${T}${Capitalize<SnakeToCamelCase<U>>}`
  : S;
```

but when I try to build using netlify I cannot modify the code in the `node_modules` and it fails the build with the following error:

```
3:00:40 PM: Error: node_modules/swell-js/types/account/camel.d.ts:1:10 - error TS2459: Module '".."' declares 'SnakeToCamelCase' locally, but it is not exported.
3:00:40 PM: 
3:00:40 PM: 1 import { SnakeToCamelCase } from '..';
3:00:40 PM:            ~~~~~~~~~~~~~~~~
3:00:40 PM: 
3:00:40 PM:   node_modules/swell-js/types/index.d.ts:54:6
3:00:40 PM:     54 type SnakeToCamelCase<S extends string> = S extends `${infer T}_${infer U}`
3:00:40 PM:             ~~~~~~~~~~~~~~~~
3:00:40 PM:     'SnakeToCamelCase' is declared here.
3:00:40 PM: 
3:00:40 PM: 
3:00:40 PM: Error: node_modules/swell-js/types/attribute/camel.d.ts:1:10 - error TS2459: Module '".."' declares 'SnakeToCamelCase' locally, but it is not exported.
3:00:40 PM: 
3:00:40 PM: 1 import { SnakeToCamelCase } from '..';
3:00:40 PM:            ~~~~~~~~~~~~~~~~
3:00:40 PM: 
3:00:40 PM:   node_modules/swell-js/types/index.d.ts:54:6
3:00:40 PM:     54 type SnakeToCamelCase<S extends string> = S extends `${infer T}_${infer U}`
3:00:40 PM:             ~~~~~~~~~~~~~~~~
3:00:40 PM:     'SnakeToCamelCase' is declared here.
3:00:40 PM: 
3:00:40 PM: 
3:00:40 PM: Error: node_modules/swell-js/types/card/camel.d.ts:1:10 - error TS2459: Module '".."' declares 'SnakeToCamelCase' locally, but it is not exported.
3:00:40 PM: 
3:00:40 PM: 1 import { SnakeToCamelCase } from '..';
```

any help? this is my init function:

```ts
    swell.init(environment.swell.storeId, environment.swell.publicApi, {
      useCamelCase: false,
    });
```

**Comment by greph** [comment]
Hi @soyzamudio , your init function looks good. We may just need to explicitly export the snakeToCamel type to resolve this issue in Angular. We'll take a look and include any necessary updates in the next release.

**Comment by soyzamudio** [comment]
@greph also notice that I had to change the `<S extends string>`  to `<S extends any>` because that was causing errors too

**Comment by soyzamudio** [comment]
I opened a PR with with the changes:

https://github.com/swellstores/swell-js/pull/89

**Comment by greph** [comment]
Thanks @soyzamudio , we're currently mirroring from Gitlab but I've pulled in and tested your changes on our end.

**Comment by soyzamudio** [comment]
is there an ETA for the next release? This is kind of blocking the release of our store and we wanted to know when we would be able to launch

**Comment by greph** [comment]
Hi @soyzamudio , we've just published 3.19.7, which includes the type update. Hope that fixes the issue for you!

**Comment by soyzamudio** [comment]
Awesome, that fixed it for me, thanks a lot @greph!
